log/format_test.go

```diff
@@ -2,120 +2,12 @@ package log
 
 import (
 	"fmt"
-	"math"
-	"math/big"
-	"math/rand"
 	"strings"
 	"testing"
 
-	"github.com/holiman/uint256"
+	"golang.org/x/exp/slog"
 )
 
-func TestPrettyInt64(t *testing.T) {
-	tests := []struct {
-		n int64
-		s string
-	}{
-		{0, "0"},
-		{10, "10"},
-		{-10, "-10"},
-		{100, "100"},
-		{-100, "-100"},
-		{1000, "1000"},
-		{-1000, "-1000"},
-		{10000, "10000"},
-		{-10000, "-10000"},
-		{99999, "99999"},
-		{-99999, "-99999"},
-		{100000, "100,000"},
-		{-100000, "-100,000"},
-		{1000000, "1,000,000"},
-		{-1000000, "-1,000,000"},
-		{math.MaxInt64, "9,223,372,036,854,775,807"},
-		{math.MinInt64, "-9,223,372,036,854,775,808"},
-	}
-	for i, tt := range tests {
-		if have := FormatLogfmtInt64(tt.n); have != tt.s {
-			t.Errorf("test %d: format mismatch: have %s, want %s", i, have, tt.s)
-		}
-	}
-}
-
-func TestPrettyUint64(t *testing.T) {
-	tests := []struct {
-		n uint64
-		s string
-	}{
-		{0, "0"},
-		{10, "10"},
-		{100, "100"},
-		{1000, "1000"},
-		{10000, "10000"},
-		{99999, "99999"},
-		{100000, "100,000"},
-		{1000000, "1,000,000"},
-		{math.MaxUint64, "18,446,744,073,709,551,615"},
-	}
-	for i, tt := range tests {
-		if have := FormatLogfmtUint64(tt.n); have != tt.s {
-			t.Errorf("test %d: format mismatch: have %s, want %s", i, have, tt.s)
-		}
-	}
-}
-
-func TestPrettyBigInt(t *testing.T) {
-	tests := []struct {
-		int string
-		s   string
-	}{
-		{"111222333444555678999", "111,222,333,444,555,678,999"},
-		{"-111222333444555678999", "-111,222,333,444,555,678,999"},
```

Comment:
Have these testcases been lifted over somewhere else? IMO we want to retain this behaviour (and some form of tests for this behaviour)

Reply:
No.  This reminded me that by default, the slog logfmt/json handlers emit output that is a different format than our current ones.  That's a missing piece that still needs to be added to this PR (as well as reintroducing test coverage for it).